aws_marketplace/curating_aws_marketplace_listing_and_sample_notebook/Algorithm/Sample_Notebook_Template/title_of_your_product-Algorithm.ipynb

```diff
@@ -0,0 +1,867 @@
+    "1. [Subscribe to the algorithm](#1.-Subscribe-to-the-algorithm)\n",
+    "1. [Prepare dataset](#2.-Prepare-dataset)\n",
+    "\t1. [Dataset format expected by the algorithm](#A.-Dataset-format-expected-by-the-algorithm)\n",
+    "\t1. [Configure and visualize train and test dataset](#B.-Configure-and-visualize-train-and-test-dataset)\n",
+    "\t1. [Upload datasets to Amazon S3](#C.-Upload-datasets-to-Amazon-S3)\n",
+    "1. [Train a machine learning model](#3:-Train-a-machine-learning-model)\n",
+    "\t1. [Set up environment](#3.1-Set-up-environment)\n",
+    "\t1. [Train a model](#3.2-Train-a-model)\n",
+    "1. [Deploy model and verify results](#4:-Deploy-model-and-verify-results)\n",
+    "    1. [Deploy trained model](#A.-Deploy-trained-model)\n",
+    "    1. [Create input payload](#B.-Create-input-payload)\n",
+    "    1. [Perform Real-time inference](#C.-Perform-Real-time-inference)\n",
+    "    1. [Visualize output](#D.-Visualize-output)\n",
+    "    1. [Calculate relevant metrics](#E.-Calculate-relevant-metrics)\n",
+    "    1. [Delete the endpoint](#F.-Delete-the-endpoint)\n",
+    "1. [Tune your model! (optional)](#5:-Tune-your-model!-(optional))\n",
+    "\t1. [Tuning Guidelines](#A.-Tuning-Guidelines)\n",
+    "\t1. [Define Tuning configuration](#B.-Define-Tuning-configuration)\n",
+    "\t1. [Run a Model Tuning job](#C.-Run-a-Model-Tuning-job)\n",
+    "1. [Perform Batch inference](#6.-Perform-Batch-inference)\n",
+    "1. [Clean-up](#7.-Clean-up)\n",
+    "\t1. [Delete the model](#A.-Delete-the-model)\n",
+    "\t1. [Unsubscribe to the listing (optional)](#B.-Unsubscribe-to-the-listing-(optional))\n",
+    "\n",
+    "\n",
+    "#### Usage instructions\n",
+    "You can run this notebook one cell at a time (By using Shift+Enter for running a cell)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Subscribe to the algorithm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To subscribe to the algorithm:\n",
+    "1. Open the algorithm listing page <font color='red'> For Seller to update:[Title_of_your_product](Provide link to your marketplace listing of your product).</font>\n",
+    "1. On the AWS Marketplace listing,  click on **Continue to subscribe** button.\n",
+    "1. On the **Subscribe to this software** page, Review and click on **\"Accept Offer\"** if you agree with EULA, Pricing, and support terms. \n",
+    "1. Once you click on **Continue to configuration button** and then choose a **region**, you will see a **Product Arn**. This is the algorithm ARN that you need to specify while training a custom ML model. Please copy the ARN corresponding to your region and specify the same in the following cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "algo_arn='<Customer to specify algorithm ARN corresponding to their AWS region>'"
```

Comment:
use seller to be consistent?

Reply:
Not really.  algo_arn needs to be specified by the customer at runtime. The goal of these templates is for seller to create a sample notebook that customers would use to try out the product from AWS Marketplace.